src/main/java/com/stripe/exception/StripeException.java

```diff
@@ -49,6 +56,12 @@ public Integer getStatusCode() {
     return statusCode;
   }
 
+  /**
+   * Returns a description of the exception, including the HTTP status code and request ID (if
+   *     applicable).
```

Comment:
Is this line supposed to be indented?

Reply:
Oops! No, indentation in Javadoc is not necessary in the summary, only when using e.g. `@param` or `@return`. Fixed!